provider/consul_catalog.go

```diff
@@ -79,8 +86,11 @@ func (provider *ConsulCatalog) healthyNodes(service string) (catalogUpdate, erro
 	}
 
 	return catalogUpdate{
-		Service: service,
-		Nodes:   data,
+		Service: &serviceUpdate{
+			ServiceName: service,
+			Attributes:  data[0].Service.Tags,
```

Comment:
tags are also available through `/services` aka `watchServices` where they are a union of all tags for the service. I'm not sure if that's in any way better than picking the first but it's perhaps a bit more consistent.